coalib/bears/requirements/PipRequirement.py

```diff
@@ -24,3 +25,29 @@ def __init__(self, package, version=""):
         :param version: A version string. Leave empty to specify latest version.
         """
         PackageRequirement.__init__(self, 'pip', package, version)
+
+    def install_command(self):
+        """
+        Creates the installation command for the instance of the class.
+
+        >>> PipRequirement('setuptools', '19.2').install_command()
+        'pip install setuptools==19.2'
+
+        :param return: A string with the installation command.
+        """
+        return "pip install {}=={}".format(self.package, self.version)
+
+    def is_installed(self):
+        """
+        Checks if the dependency is installed.
+
+        >>> PipRequirement('pip').is_installed()
+        True
+
+        >>> PipRequirement('some_package').is_installed()
+        False
+
+        :param return: True if dependency is installed, false otherwise.
+        """
+        return not subprocess.call(['pip', 'show', self.package],
+                                   stdout=subprocess.DEVNULL)
```

Comment:
Why do we care about stderr? its not shown on the screen. I've placed it on the Npm one, as there it is.